src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -373,6 +477,18 @@ public void Patch(TStructuralType original)
             CopyChangedValues(original);
         }
 
+        /// <summary>
+        /// Overwrites the <paramref name="original"/> entity with the changes tracked by this Delta.
+        /// <remarks>The semantics of this operation are equivalent to a HTTP PATCH operation, hence the name.</remarks>
+        /// </summary>
+        /// <param name="original">The entity to be updated.</param>
+        /// <param name="patchHandler">Patch Handler</param>
+        public void Patch(TStructuralType original, IPatchMethodHandler patchHandler)
```

Comment:
>Patch [](start = 20, length = 5)

Why do we have both Patch and Copy ChangedValues?  Should we remove CopyChangedValues in 8.0?

Reply:
I beleive makingcopychangedvalues as a private method is fine in 8.0